src/renderers/shared/shared/__tests__/ReactEmptyComponent-test.js

```diff
@@ -246,14 +246,20 @@ describe('ReactEmptyComponent', () => {
     expect(assertions).toBe(3);
   });
 
-  it('throws when rendering null at the top level', () => {
-    // TODO: This should actually work since `null` is a valid ReactNode
+  it('can render null at the top level', () => {
     var div = document.createElement('div');
-    expect(function() {
+
+    if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
Is @spicyj ok with this divergence? Seems like it would be pretty easy to start relying on this, even accidentally.